assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSelectionPanel/AudienceItem.js

```diff
@@ -108,6 +114,25 @@ export default function AudienceItem( {
 
 	const id = `audience-selection-checkbox-${ slug }`;
 
+	function ItemBadge() {
+		if ( temporarilyHidden ) {
+			return (
+				<BadgeWithTooltip
+					className="googlesitekit-audience-segmentation-temporary-hidden-badge"
+					label={ __( 'Temporarily hidden', 'google-site-kit' ) }
+					tooltipTitle={ __(
+						'Site Kit is collecting data for this group. Once data is available the this group will be added to your dashboard.',
```

Comment:
As per @nfmohit's comment this needs a grammatical fix. I'll apply the change myself to move this along.

```suggestion
						'Site Kit is collecting data for this group. Once data is available the group will be added to your dashboard.',
```